tests/rptest/tests/compatibility/kafka_streams_test.py

```diff
@@ -142,24 +142,25 @@ def try_cons():
         example.stop()
 
 
-class KafkaStreamsTopArticles(KafkaStreamsDriverBase):
-    """
-    Test KafkaStreams TopArticles which counts the top N articles
-    from a stream of page views
-    """
-    topics = (
-        TopicSpec(name="PageViews"),
-        TopicSpec(name="TopNewsPerIndustry"),
-    )
-
-    Example = KafkaStreamExamples.KafkaStreamsTopArticles
-    Driver = KafkaStreamExamples.KafkaStreamsTopArticles
-
-    def __init__(self, test_context):
-        super(KafkaStreamsTopArticles,
-              self).__init__(test_context=test_context,
-                             enable_pp=True,
-                             enable_sr=True)
+# Disabled for https://github.com/redpanda-data/redpanda/issues/4637
```

Comment:
Wrong test to disable.
It should be `KafkaStreamsPageView`

Reply:
:man_facepalming: fixed.